frontend/src/hooks/api/getters/useFeatureSearch/useFeatureSearch.ts

```diff
@@ -55,17 +55,19 @@ const createFeatureSearch = () => {
     return (
         params: SearchFeaturesParams,
         options: SWRConfiguration = {},
+        cachePrefix: string = '',
     ): UseFeatureSearchOutput => {
         const { KEY, fetcher } = getFeatureSearchFetcher(params);
```

Comment:
is the KEY in use here after this change?

Reply:
Yes, on line 61.

Reply:
Sigh, blink and you miss it 😅